catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/view/Registry.view.js

```diff
@@ -0,0 +1,227 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+define([
+    'icanhaz',
+    'marionette',
+    'underscore',
+    'jquery',
+    'q',
+    'js/view/ModalRegistry.view.js',
+    'js/view/EmptyView.js',
+    'wreqr',
+    'js/view/Utils.js',
+    'text!templates/deleteRegistryModal.handlebars',
+    'text!templates/deleteRegistry.handlebars',
+    'text!templates/registryPage.handlebars',
+    'text!templates/registryList.handlebars',
+    'text!templates/registryRow.handlebars'
+],
+function (ich,Marionette,_,$,Q,ModalRegistry,EmptyView,wreqr,Utils,deleteRegistryModal,deleteRegistry,registryPage,registryList,registryRow) {
+
+    var RegistryView ={};
+
+    ich.addTemplate('deleteRegistryModal', deleteRegistryModal);
+    ich.addTemplate('deleteRegistry', deleteRegistry);
+    ich.addTemplate('registryPage', registryPage);
+    ich.addTemplate('registryRow', registryRow);
+    ich.addTemplate('registryList', registryList);
+
+    RegistryView.RegistryRow = Marionette.Layout.extend({
+        template: "registryRow",
+        tagName: "tr",
+        className: "highlight-on-hover",
+        regions: {
+            editModal: '.modal-container'
+        },
+        events: {
+            'click td' : 'editRegistry'
+        },
+        initialize: function(){
+            _.bindAll(this);
+            this.listenTo(this.model, 'change', this.render);
+            this.listenTo(wreqr.vent, 'status:update-'+ this.model.attributes.pid, this.updateStatus);
```

Comment:
This is the wrong way to use an attribute from a model, you should be doing this.model.get('pid'). Also, you could remove wreqr from the model and just do `this.listenTo(this.model, 'sync', this.updateStatus)`

Or you could add a modelEvents object to this view and simply do something like:
```
modelEvents: {
    'sync' : 'updateStatus'
}
```